sio-digital-team/active-safe
# Prevent section

[Trello card]( https://trello.com/c/vhbWgnWr/60-build-the-new-prevent-mockup-35)

**WIP**

**Changes:**

- [x] removed unnecessary slick code
- [x] removed images and used SVGs instead
- [x] added prevent section for desktop
- [x] added prevent section for mobile

  
**How to test:**

- go to homepage, scroll down to prevent section
- on desktop, content should open on right
- on mobile, content should open like accordion
  

**Todo:**

- [ ] mobile media querie doesn't stay on mobile
- [ ] @kevin-quach's help with media querie
